### PR TITLE
Review changes for compare list

### PR DESCRIPTION
**Pull request by akaplya**

### CODE REVIEW

File: design-documents/graph-ql/coverage/catalog/compare-list.graphqls [hunk 1]
```diff
@@ -36,7 +36,7 @@ type Mutation {
         id: ID!
         items: [ID!]
     ): CompareList
-    assignCompareListToCustomer(customerId: ID!, listId: ID!): Boolean
+    assignCompareListToActiveCustomer(listId: ID!): Boolean
```
Comment (paliarush) [inline design-documents/graph-ql/coverage/catalog/compare-list.graphqls]: Usually we do not use 'active' in the name, and assume it

**Comment by nrkapoor** [comment]
@akaplya Can you please review this PR?

**Comment by nrkapoor** [comment]
@melnikovi @akaplya Can we merge this PR?

File: design-documents/graph-ql/coverage/catalog/compare-list.graphqls (hunk 1 above)
Comment (melnikovi) [inline design-documents/graph-ql/coverage/catalog/compare-list.graphqls:39]:
```suggestion
    assignCompareListToCustomer(listId: ID!): Boolean
```

**Review by melnikovi — APPROVED** [review]

**Pull request merged**